recipes/tempeh/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "tempeh" %}
+{% set version = "0.1.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 9565fbeb2bef5b23f3aa5412fa33956b7df520d294e27d403f1e3a02e3a26b65
+
+build:
+  number: 0
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
  number: 0
```